packages/breakpoint-split-view/src/BreakpointSplitView.ts

```diff
@@ -23,15 +23,11 @@ export default ({ jbrequire }: { jbrequire: Function }) => {
       let endPos
       const bpPerPx = 10
 
+      // TODO: Figure this out for multiple assembly names
```

Comment:
What needs to be done here?

Reply:
This is actually mostly unrelated to the assembly manager and was a bugfix for looking up canonical refNames in breakpoint split view. This comment should have been there since we got rid of displayRegionsFromAssemblyName, but it got missed.

Basically the idea is that you get a feature, and you want to get the canonical refName for it. To do that, you need an assemblyName, which it just grabs from the first displayedRegion on the view. But what if the view has displayedRegions from multiple assemblies, how do you know which one to use to look up the refName? An odd case, but one that's technically supported.

There are similar comments for similar cases in packages/circular-view/src/StructuralVariantChordTrack/models/refNameMapKeeper.js and packages/wiggle/src/WiggleTrack/model.ts